.github/workflows/kubernetes-deploy.yaml

```diff
@@ -81,6 +78,7 @@ jobs:
           echo $GATEWAY_HOST
           # basic tests
           cd /home/runner/work/qiskit-serverless/qiskit-serverless/tests/basic
+          rm 06_function.py
```

Comment:
nit: should we put the function test in its own directory to prevent running it?